src/shared/locales/en/translation.json

```diff
@@ -737,6 +737,10 @@
          "setupIncomplete": "Wallet setup incomplete",
          "completeSetup": "Complete setup"
     },
+    "proxy": {
+        "proxy": "Use system proxy",
+        "proxyExplanation": "Detect and use system wide network proxy settings. Restart the wallet after setting change."
```

Comment:
Maybe this should be changed to `Restart the wallet after changing this setting`?